code/modules/paperwork/faxmachine.dm

```diff
@@ -0,0 +1,60 @@
+
+//The IA Agent's Fax Machine
+
+/obj/machinery/faxmachine
+	name = "fax machine"
+	desc = "Used to send messages to your corporate overlords (and promptly get fired for writing fanfiction about their mothers)."
+	circuit = /obj/item/circuitboard/machine/faxmachine
+	icon = 'icons/obj/library.dmi'
+	icon_state = "fax"
+	density = TRUE
```

Comment:
Most of these vars are already the same on the parent, don't need to redefine them for nothing.